From mrobbins@MIT.EDU  Mon May 19 17:07:10 2008
From: "Michael F. Robbins" <mrobbins@MIT.EDU>
Date: Fri, 16 May 2008 23:48:42 -0400
Subject: USB: serial: ch341: New VID/PID for CH341 USB-serial
To: gregkh@suse.de
Cc: hevans@MIT.EDU, frank@kingswood-consulting.co.uk
Message-ID: <1210996122.3530.13.camel@maximum-entropy.mit.edu>


Recent USB-serial devices using the WinChipHead CH340/CH341 chipset are
being shipped with a new vendor/product ID code pair, but an otherwise
identical device.  (This is confirmed by looking at INF for the included
Windows driver.)

Patch is tested and working, both with new and old devices.

Signed-off-by: Michael F. Robbins <mrobbins@mit.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/ch341.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/ch341.c
+++ b/drivers/usb/serial/ch341.c
@@ -28,6 +28,7 @@ static int debug;
 
 static struct usb_device_id id_table [] = {
 	{ USB_DEVICE(0x4348, 0x5523) },
+	{ USB_DEVICE(0x1a86, 0x7523) },
 	{ },
 };
 MODULE_DEVICE_TABLE(usb, id_table);
